src/App.js

```diff
@@ -37,7 +41,11 @@ const tweetsArray = [
 function App() {
   return (
     <div className="App">
-      <Tweet />
+    <NavBar />
+    <h1 className="IronTitle">Iron Tweet</h1>
+      <Tweet tweet={ tweetsArray[0] }/>
```

Comment:
Si tuvieramos un array con 100 elementos esto no es tan rentable en su lugar podriamos hacer un map para iterar el array 👀 
```suggestion
{tweetsArray.map((tweet, i) => (
        <Tweet key={i} tweet={tweet} />
      ))}```